qualtran/bloqs/mcmt/and_bloq.py

```diff
@@ -260,26 +256,38 @@ class MultiAnd(Bloq):
     Args:
         cvs: A tuple of control variable settings. Each entry specifies whether that
             control line is a "positive" control (`cv[i]=1`) or a "negative" control `0`.
+            If a (symbolic) integer is passed, assumes the control values to be all 1's.
+
 
     Registers:
         ctrl: An n-bit control register.
         junk [right]: An `n-2` bit junk register to be cleaned up by the inverse operation.
         target [right]: The output bit.
     """
 
-    cvs: Tuple[Union[int, sympy.Expr], ...] = field(converter=_to_tuple)
+    cvs: Union[Length, Tuple[SymbolicInt, ...]] = field(converter=_to_tuple)
 
     @cvs.validator
     def _validate_cvs(self, field, val):
-        if len(val) < 3:
+        if not is_symbolic(val) and len(val) < 3:
             raise ValueError("MultiAnd must have at least 3 control values `cvs`.")
 
+    @property
+    def bitsize(self) -> SymbolicInt:
+        return self.cvs.bitsize if isinstance(self.cvs, Length) else len(self.cvs)
+
+    @property
+    def control_values(self) -> Tuple[SymbolicInt, ...]:
+        if isinstance(self.cvs, Length):
+            raise ValueError(f"{self.cvs} is symbolic")
+        return self.cvs
```

Comment:
cvs

Reply:
I still think this should be meaningfully disambiguated. `and_bloq.cvs` vs `and_bloq.control_values` ... 

maybe `concrete_cvs`? idk

Reply:
we have `is_symbolic` in the symbolic utils. What's the opposite of symbolic? we should pick a word for it and use that word consistently throughout the library

Reply:
I can use whatever you suggest -- I trust your expertise on the English language more than mine :)

Reply:
let's go with "concrete" https://en.wiktionary.org/wiki/concrete